ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -289,6 +304,13 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="..\packages\QuantConnect.pythonnet.1.0.5.7\build\QuantConnect.pythonnet.targets" Condition="Exists('..\packages\QuantConnect.pythonnet.1.0.5.7\build\QuantConnect.pythonnet.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
```

Comment:
nuget warning message in the csproj file?

Reply:
Those lines are present in all Lean `csproj` files where `pythonnet` is needed as dependency.
[Launcher](https://github.com/QuantConnect/Lean/blob/418b25eddc46813fc2855f49b06adac8dc54d148/Launcher/QuantConnect.Lean.Launcher.csproj#L143), [Algorithm](https://github.com/QuantConnect/Lean/blob/418b25eddc46813fc2855f49b06adac8dc54d148/Algorithm/QuantConnect.Algorithm.csproj#L130), [Common](https://github.com/QuantConnect/Lean/blob/418b25eddc46813fc2855f49b06adac8dc54d148/Common/QuantConnect.csproj#L613), etc.

Reply:
interesting!